float64 base
float64 shoulder
float64 elbow
float64 wrist1
float64 wrist2
float64 wrist3
